lib/board.rb

```diff
@@ -1,5 +1,7 @@
 class Board
   attr_reader :board
+  attr_reader :POSITION_MIN
+  attr_reader :POSITION_MAX
```

Comment:
Unfortunately :board, :POSITION_MIN and :POSITION_MAX are used in others class. They are mostly used in my tests and the interface (display the range a played can set for a move).

In another hand :counter could be private as it's a private variable, but I still didn't make it work. I had this error `undefined method `+' for nil:NilClass` when I try to do an incrementation to this variable that I set private (with an attr_accessor)

``` ruby 
 13   def set_mark(mark, position)
 14     begin
 15       position = Integer(position)
 16       valid?(position)
 17       free?(position)
 18     rescue
 19       raise
 20     end
 21
 22     board[get_row(position)][get_column(position)] = mark
 23     counter += 1
 24   end
```
I feel like I really missed a point with the way private/public work with Ruby.